sklearn/model_selection/_split.py

```diff
@@ -882,6 +908,32 @@ def get_n_splits(self, X=None, y=None, groups=None):
         groups = check_array(groups, ensure_2d=False, dtype=None)
         return len(np.unique(groups))
 
+    def split(self, X, y=None, groups=None):
+        """Generate indices to split data into training and test set.
+
+        Parameters
+        ----------
+        X : array-like, shape (n_samples, n_features)
+            Training data, where n_samples is the number of samples
+            and n_features is the number of features.
+
+        y : array-like, of length n_samples
```

Comment:
same here

Reply:
Whoops, missed that one. Should be fixed